.chronus/changes/vscode-bundle-init-2025-2-11-18-46-5.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@typespec/compiler"
+---
+
+Fix the label for @typespec/http-server-js in init template
```

Comment:
```suggestion
---
changeKind: internal
packages:
  - "@typespec/compiler"
---

```

doesn't feel like a change that's needed to appear in the release notes/changelog